docs/en/metrics/metrics-installation.asciidoc

```diff
@@ -1,100 +1,104 @@
 [[install-metrics-monitoring]]
 [role="xpack"]
-== Install Metrics
+== Get started
 
-The easiest way to get started with Elastic Logs is by using our hosted {es} Service on Elastic Cloud.
-The {es} Service is available on both AWS and GCP, and automatically configures {es} and {kib}.
+To use the {metrics-app}, you need {es} for storing and searching your data, and {kib}
+(version 6.5 or later) for visualizing and managing it.
 
-NOTE: If your data uses nonstandard fields, you may need to modify some default <<configure-metrics-source,configuration settings>>.
+To ingest data, you can use {metricbeat} installed on each server you want to monitor, or
+third-party collectors that are configured to ship ECS-compliant data. The <<metrics-fields-reference>>
+provides a list of all fields used in the {metrics-app}. 
 
-[float]
-=== Hosted Elasticsearch Service
-
-Skip installing and managing your own {es} and {kib} instance by using our hosted {es} Service.
-{ess-trial}[Try out the {es} Service for free].
+If your data uses nonstandard fields, you may need to modify some default <<configure-metrics-source,configuration settings>>.
```

Comment:
I don't think users will understand what you mean when you say "if your data uses nonstandard fields". From a user's perspective, at this point, they just have systems that they  want to monitor. Fields don't exist until after they've installed Metricbeat and data starts flowing to Elasticsearch. I think you're talking about custom index pattern/names and custom fields, but that's not clear here.

Reply:
Yes, this relates to custom index patterns and non-default fields. I moved this paragraph to a note statement in Step 4.